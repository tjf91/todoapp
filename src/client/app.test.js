import App from './app'
import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json';


describe('App',()=>{
	test('should match snapshot',()=>{
		const wrapper = shallow (<App />)

		expect(wrapper.find('div').text()).toBe("Welcome to React Boilerplate App")
		expect(wrapper).toMatchSnapshot()
	})
})
