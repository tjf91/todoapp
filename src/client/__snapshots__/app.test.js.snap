// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  Welcome to React Boilerplate App
</div>
`;
